logging/nox.py

```diff
@@ -73,7 +73,7 @@ def lint(session):
     Returns a failure if flake8 finds linting errors or sufficiently
     serious code quality issues.
     """
-    session.interpreter = 'python3.6'
+    session.interpreter = 'python3.5'
```

Comment:
Yes. Please keep it at 3.6 (it concerns me if there is a failure in 3.6 that passes in 3.5. If the answer is because you don't have 3.6 installed locally ping me and I can walk you through some "recommended" ways to install)